exercicios/ex7/action.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+function returnBook($book, $choose) {
+    $book = $_POST['book'];
+    $choose = $_POST['choose'];
+
+    if ($choose == 'prof') {
+        $result = "Você tem 10 dias pra devolver o livro $book";
+    } else {
+        $result = "Você tem 3 dias pra devolver o livro $book";
+    }
```

Comment:
Vamos fazer esse if diferente? Vamos calcular a data que pegou o livro + 10 dias, fazer um calculo de dias em php mesmo e exibir a data.

ou seja se for professor soma 10 dias se não soma 3 e retorna somente a data como resultado da função